auacm/angular/src/app/competitions/competitions.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="margin-top col-md-10 col-md-offset-1">
+  <div class="panel panel-default" *ngIf="competitions.ongoing.length > 0">
+    <div class="panel-heading">
+      <h2 class="panel-title">Ongoing</h2>
+    </div>
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th class="col-md-3">Name</th>
+          <th class="col-md-3">Start Time</th>
+          <th class="col-md-2">Length</th>
+          <th [ngClass]="user.loggedIn ? 'col-md-2' : 'col-md-4'">Remaining</th>
+          <th class="col-md-2" *ngIf="user.loggedIn">Register</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr *ngFor="let competition of competitions.ongoing | orderBy:'startTime'">
```

Comment:
I personally think it would be better to put the most recent contest at the top.
Thoughts?